source/configuration_manual/namespace/index.rst

```diff
@@ -32,6 +32,182 @@ you to update an existing namespace (like how ``15-mailboxes.conf`` does) or
 have userdb override namespace settings for specific users
 ``namespace/sectionname/prefix=foo/``.
 
+All namespace settings with defaults::
+
+  namespace inbox {
+    disabled = no
+    hidden = no
+    ignore_on_failure = no
+    inbox = no
+    list = yes
+    location =
+    order = 0
+    prefix =
+    separator =
+    subscriptions = yes
+    type = private
+  }
+
+
+.. _setting-namespace_name:
+
+``namespace name``
+------------------
+
+- Default: <empty>
+- Values: :ref:`string`
+
+Name of the namespace.
+Giving name is optional, but doing so enables referencing the configuration later on.
+
+
+.. _setting-namespace_disabled:
+
+``namespace/disabled``
+----------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace is disabled, and cannot be accessed by user in any way.
+
+
+.. _setting-namespace_hidden:
+
+``namespace/hidden``
+--------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace will be hidden from user.
+It is still accessible if folders are directly opened, or referenced by other components.
+Usually used for lazy expunge or virtual folders.
```

Comment:
More specifically: This hides the namespace from the IMAP NAMESPACE command response. Otherwise it doesn't change anything, i.e. the namespace is still in LIST reply.